_sass/pages/_repl.scss

```diff
@@ -194,6 +194,8 @@
 
 .babel-repl-v2 {
   height: 100%;
+  margin-bottom: $padding-large-vertical * -4; // Compensate for _footer.scss margin-top
```

Comment:
Why `-4`? Is there a constant for that somewhere that you can reuse?

The current REPL has the footer hanging off the bottom of the page: https://babeljs.io/repl/. I think we should just keep that same style.